__tests__/app.test.js

```diff
@@ -0,0 +1,18 @@
+const request = require('supertest')
+const app = require('../app')
+const data = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+
+const categoriesData = require('../db/data/test-data/categories')
+
+beforeEach(() => {
+    return seed(data);
+})
+
+describe('GET Categories', () => {
+    test('should return object with one entry: {"categories" : <all categories data>}', () => {
+        return request(app).get('/api/categories')
+        .expect(200)
+        .expect({'categories' : categoriesData})
```

Comment:
You don't want to be testing like this as it is brittle. Should your categories data change order the test will fail. It's similar to saying an object should equal an object, it's not extensible. 
You're also not utilising Jest or ending the connection to the db, i'm not sure if that will be automatically done by supertest. Great creativity but refactor needed 😀

Reply:
Actually just had a conversation about this during an nchelp haha. I will fix it